cfme/tests/containers/test_basic_metrics.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from utils import testgen
+from utils import conf
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda provider: current_version() < "5.6"),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope='function')
+
+
+def test_basic_metrics(provider, ssh_client):
+    """ Basic Metrics availability test
+        This test checks that the Metrics service is up
+        Curls the hawkular status page and checks if it's up
+        """
+    username, password = provider.credentials['token'].principal,\
+        provider.credentials['token'].secret
+    hostname = conf.cfme_data.get('management_systems', {})[provider.key]\
+        .get('hostname', [])
```

Comment:
@jaryn I see no circumstances under which the hostname will not be set. That value comes from the config yamls. No way it's left empty, it will fail all the tests run